crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -712,7 +710,7 @@ module sui::sui_system {
         validator_set::staking_pool_mappings(&self.validators)
     }
 
-    /// Returns all the validators who have reported `addr` this epoch.
+    /// Returns all the validators who are currently reporting `addr`
     public fun get_reporters_of(wrapper: &SuiSystemState, addr: address): VecSet<address> {
         let self = load_system_state(wrapper);
         if (vec_map::contains(&self.validator_report_records, &addr)) {
```

Comment:
Now that the records stay across epochs, things get a bit more complicated, since reporters can leave the validator set and their scores should no longer count.

Reply:
We should probably clean up validator_report_records each epoch to rid of the inactive validators' votes.

Reply:
Not for this PR, but long-term do we need to worry about validators rotating their address? If so, this could change how we want to record/clean these votes I think (obj-id vs address based).